eng/pipelines/templates/jobs/smoke-tests.yml

```diff
@@ -0,0 +1,135 @@
+          ArmTemplateParameters: $(azureCloudArmParameters)
+        Linux (AzureCloud Canary):
+          OSVmImage: "ubuntu-18.04"
+          TestTargetFramework: netcoreapp2.1
+          SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
+          ArmTemplateParameters: $(azureCloudArmParameters)
+          Location: "eastus2euap"
+        Windows_NetCoreApp (AzureCloud):
+          OSVmImage: "windows-2019"
+          TestTargetFramework: netcoreapp2.1
+          SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
+          ArmTemplateParameters: $(azureCloudArmParameters)
+        Windows_NetFramework (AzureCloud):
+          OSVmImage: "windows-2019"
+          TestTargetFramework: net461
+          SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
+          ArmTemplateParameters: $(azureCloudArmParameters)
+        MacOs (AzureCloud):
+          OSVmImage: "macOS-10.15"
+          TestTargetFramework: netcoreapp2.1
+          SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
+          ArmTemplateParameters: $(azureCloudArmParameters)
+        Windows_NetCoreApp (AzureUSGovernment):
+          OSVmImage: "windows-2019"
+          TestTargetFramework: netcoreapp2.1
+          SubscriptionConfiguration: $(sub-config-gov-test-resources)
+          ArmTemplateParameters: $(azureUSGovernmentArmParameters)
+        Windows_NetFramework (AzureUSGovernment):
+          OSVmImage: "windows-2019"
+          TestTargetFramework: net461
+          SubscriptionConfiguration: $(sub-config-gov-test-resources)
+          ArmTemplateParameters: $(azureUSGovernmentArmParameters)
+        Windows_NetCoreApp (AzureChinaCloud):
+          OSVmImage: "windows-2019"
+          TestTargetFramework: netcoreapp2.1
+          SubscriptionConfiguration: $(sub-config-cn-test-resources)
+          ArmTemplateParameters: $(azureChinaCloudArmParameters)
+        Windows_NetFramework (AzureChinaCloud):
+          OSVmImage: "windows-2019"
+          TestTargetFramework: net461
+          SubscriptionConfiguration: $(sub-config-cn-test-resources)
+          ArmTemplateParameters: $(azureChinaCloudArmParameters)
+
+    variables:
+      azureCloudArmParameters: "@{ storageEndpointSuffix = 'core.windows.net'; azureCloud = 'AzureCloud'; }"
+      azureUSGovernmentArmParameters: "@{ storageEndpointSuffix = 'core.usgovcloudapi.net'; azureCloud = 'AzureUSGovernment'; }"
+      azureChinaCloudArmParameters: "@{ storageEndpointSuffix = 'core.chinacloudapi.cn'; azureCloud = 'AzureChinaCloud'; }"
+
+    pool:
+      vmImage: $(OSVmImage)
+
+    steps:
+      - template: /eng/common/pipelines/templates/steps/verify-agent-os.yml
+
+      - task: PowerShell@2
+        ${{ if eq(parameters.Nightly, true) }}:
+          displayName: Use latest dev feed package versions
+        ${{ if eq(parameters.Nightly, false) }}:
+          displayName: Use latest package versions
```

Comment:
latest GA? Or does this also include beta?

Reply:
edit: Sorry I mean this should include "alpha" I might have misspoken similarly in a couple other comments as well. alpha is the new dev/daily/nightly built version. beta is for public previews.

Reply:
It will include beta and potentially alpha if there is no ga/beta.